bin/prover-client/src/errors.rs

```diff
@@ -1,46 +1,46 @@
+use strata_db::DbError;
+use strata_primitives::proof::ProofKey;
+use strata_zkvm::ZkVmError;
 use thiserror::Error;
 
+use crate::status::ProvingTaskStatus;
+
 // Define custom error type
 #[derive(Error, Debug)]
 pub enum ProvingTaskError {
```

Comment:
This comment needs to be expanded into a docstring.
Also can you add docstrings to the variants?

Reply:
I know that the error messages are really helpful, but they don't show up in the LSP (IDE autocompletions) or documentation.